### DISCUSSION TITLE
Unable to dl from Bigmarker

### DISCUSSION THREAD

**Discussion by User74923** [aclap-dev/video-downloadhelper#1472] [Help and Q&A]
### Your question

Hello, 
This site was working a couple of weeks ago but not now. The video I get has the first few seconds then it goes blank.

**Comment by Mattmjs** [comment]
This is rather surprising, it is using DRM protection without the need to run DRM in the browser. I have it turned off in Firefox.
Unfortunately downloading is not going to work for your example or maybe the site overall.
